examples/ads.amp.html

```diff
@@ -557,6 +558,17 @@ <h2 id="ezoic">Ezoic</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="felmat">felmat</h2>
```

Comment:
I am not able to see a test ad with your example provided. I would highly encourage you to update the example here to provide a render-start ad as it helps with testing your PR this time, and manual test later on.